### DISCUSSION TITLE
X-axis combining two variables

### DISCUSSION THREAD

**Discussion by anders-kruse** [vega/vega-lite#8449] [General]
I want two make a chart combining two variables in the x-axis, like the excel-example under. Is that possible?

I'm new to vega-lite and I can't find any examples doing this. Thank you very much!

![image](https://user-images.githubusercontent.com/52952511/194034035-5b58e50d-9195-480a-aa61-a4ab78039430.png)

**Comment by jwoLondon** [comment]
You can create a grouped bar chart with `xOffset` positions. There's an example here:  https://vega.github.io/vega-lite/docs/encoding.html#positon-offset

You may also need to use conditional axis formatting. There's an example in the documentation here: https://vega.github.io/vega-lite/docs/axis.html#example-conditional-axis-properties-and-multi-line-axis-label

**Reply by anders-kruse** [reply]
That makes sense! Thank you very much, jwoLondon
